tools/depends/target/samba-gplv3/Makefile

```diff
@@ -35,6 +35,10 @@ CONFIGURE=./configure --prefix=$(PREFIX) \
           --without-regedit \
           --without-lttng
 
+ifeq ($(TARGET_PLATFORM), webos)
+  CONFIGURE += --with-libiconv=$(PREFIX)/lib
+endif
+
```

Comment:
Im thinking maybe this should be opened up to more platforms.

probably just use NEED_LIBICONV as the check, as that means a platform has built iconv as a dependency

https://github.com/xbmc/xbmc/blob/5709641d55b7d49167a7e52573c542c11f523fee/tools/depends/target/Makefile#L86

Reply:
I was thinking the same if this was also an issue on android. Updated the patch to check for `NEED_LIBICONV` instead.